core/templates/components/state-editor/state-editor.directive.html

```diff
@@ -1,4 +1,4 @@
-<md-card class="oppia-editor-card-with-avatar">
+<md-card class="oppia-editor-card-with-avatar oppia-editor-card-with-avatar-sticky">
```

Comment:
Can we pass in a boolean input for state-editor that will determine if the "sticky" functionality should be enabled (e.g. in the question editor, add sticky and in other places don't add)? The current setup seems to define CSS in a parent component but gets used only in the child component -- this does not match the current pattern in the codebase i.e. CSS related to component X is defined in the template HTML for X. Not adhering to this established pattern for the component in question, can result in regressions because devs may remove CSS in the parent scope thinking that it is not used anywhere.